remote_vector_index_builder/core/common/models/index_builder/response/faiss_index_id_map.py

```diff
@@ -0,0 +1,40 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import faiss
+from dataclasses import dataclass
+
+
+@dataclass
+class FaissIndexIDMap:
```

Comment:
I didn't think name of the class will this specific. Can we name it BuildIndexOutput.